chrome-extension/public/popup.html

```diff
@@ -0,0 +1,6 @@
+<html>
+  <body>
+    <h1>Random Image of the Moment:</h1>
+    <img src="https://source.unsplash.com/random" alt="" width="650px" height="500px"/>
```

Comment:
it's always good to have something defined in the `alt` attribute for images. something as simple as `alt="photo"` would be good just so it isn't empty